dawn/src/dawn/Unittest/CompilerUtil.cpp

```diff
@@ -155,6 +157,9 @@ CompilerUtil::createGroup(PassGroup group, std::unique_ptr<OptimizerContext>& co
     addPass<dawn::PassMultiStageSplitter>(context, passes, mssSplitStrategy);
     addPass<dawn::PassStageSplitter>(context, passes);
     addPass<dawn::PassTemporaryType>(context, passes);
+    addPass<dawn::PassLocalVarType>(context, passes);
+    addPass<dawn::PassRemoveScalars>(context, passes);
+    // TODO: how to add grid-dependent passes?: PassStageSplitAllStatements, PassSetLocationType
```

Comment:
Still not clear what is the intended way to use this method (it's not used yet).
Does it aim at replacing also the code doing the same thing in DawnCompiler.cpp?